### ISSUE TITLE
Add Activities

### ISSUE THREAD

**Issue by oceansize** [makersacademy/pipekit-webmock#3]
To support the Activities feature in Pipekit, I must therefore add it here so we can run our tests properly.

**Comment by oceansize** [comment]
Pipekit-webmock actually speaks to the newly-added `ResourceLabel` object, which put a wrench in plans. Used a Struct, which worked but [seemed a bit of an ugly solution](https://github.com/makersacademy/pipekit-webmock/commit/5679a74259b4af1bc6b1f163ba37ceb66035a9c4)

**Comment by oceansize** [comment]
[In this commit](https://github.com/makersacademy/pipekit-webmock/commit/497a2eb520f2b6574699488097235111ed32246f) I refactored the struct and hash table out of the API and into it's own object, which seemed cleaner. However, there was then a clash with the original `ResourceLabel` once I moved that code into a class. The simplest fix here was to rename the test version to `ResourceName`, which still feels a bit funky, but seemed like the best implementation for now.
